app/views/users/show.html.erb

```diff
@@ -1,7 +1,13 @@
-<h1><%= @user.name %></h1>
+
+<div class="show-user">
+<div class="container"><div class="cadre"></div><div class="user"><h2><%= @user.name  %></h2> <p><span></span><span>Number of posts : <span><%= @user.posts_counter %></span></span></p></div></div>
+
+<p class='bio'><%= @user.bio %></p>
 <br>
-<p><%= @user.bio %></p>
+<% @post.each_with_index do |posts, idx|%>
+    <div class='post'><span class='title'><%= link_to 'Post #', user_posts_path(@user) %><%= idx + 1 %></span>
+    <br><%= posts.text %></div>
+<% end %>
```

Comment:
- [ ] Kindly make sure to use the `#recent_post` method to render the user's 3 most recent posts to match the project's requirements.